app/code/Magento/MediaContent/etc/db_schema.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<!--
+/**
+ * Copyright © Magento, Inc. All rights reserved.
+ * See COPYING.txt for license details.
+ */
+-->
+<schema xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="urn:magento:framework:Setup/Declaration/Schema/etc/schema.xsd">
+    <table name="media_content_asset" resource="default" engine="innodb" comment="Relation between media content and media asset">
+        <column xsi:type="int" padding="10" name="asset_id" unsigned="true" nullable="false" identity="true" comment="Entity ID"/>
+        <column xsi:type="varchar" length="255" name="entity_type" nullable="false" comment="Content type"/>
+        <column xsi:type="varchar" length="255" name="entity_id" nullable="false" comment="Content entity id"/>
+        <column xsi:type="varchar" length="255" name="field" nullable="false" comment="Content field"/>
+        <constraint xsi:type="primary" referenceId="PRIMARY">
+            <column name="asset_id"/>
```

Comment:
`asset_id` should be removed from the PRIMARY key. The index should be created `asset_id`